ui/components/app/wallet-overview/coin-overview.tsx

```diff
@@ -136,29 +241,93 @@ export const CoinOverview = ({
+                      style={{ display: 'contents' }}
+                      className={classnames(
+                        `${classPrefix}-overview__primary-balance`,
+                        {
+                          [`${classPrefix}-overview__cached-balance`]:
+                            balanceIsCached,
+                        },
+                      )}
+                      data-testid={`${classPrefix}-overview__primary-currency`}
+                      value={balanceToDisplay}
+                      type={PRIMARY}
+                      ethNumberOfDecimals={4}
+                      hideTitle
+                      shouldCheckShowNativeToken
+                      isAggregatedFiatOverviewBalance={
+                        !showNativeTokenAsMainBalance && !isTestnet
+                      }
+                    />
+                  </Box>
+                  {shouldShowPopover &&
+                  (!isTestnet || (isTestnet && showFiatInTestnets)) &&
+                  !showNativeTokenAsMainBalance ? (
+                    <Popover
+                      referenceElement={referenceElement}
+                      isOpen={isOpen}
+                      position={PopoverPosition.BottomStart} // TODO check with design-team about this bottom start issue
+                      hasArrow
+                      flip
+                      data-theme={theme === 'light' ? 'dark' : 'light'}
+                      className="balance-popover__container"
+                      width={BlockSize.Full}
+                      padding={3}
+                      onClickOutside={handleClick}
+                      onPressEscKey={handleClick}
+                      preventOverflow
+                    >
+                      <Box>
+                        <Box
+                          display={Display.Flex}
+                          justifyContent={JustifyContent.spaceBetween}
+                        >
+                          <Text
+                            variant={TextVariant.bodySmBold}
+                            textAlign={TextAlign.Left}
+                            alignItems={AlignItems.flexStart}
+                          >
+                            {t('yourBalanceIsAggregated')}
+                          </Text>
+                          <ButtonIcon
+                            size={ButtonIconSize.Sm}
+                            onClick={handleClick}
+                            iconName={IconName.Close}
+                            justifyContent={JustifyContent.center}
+                            ariaLabel="close"
+                            data-testid="popover-close"
+                          />
+                        </Box>
+
+                        <Text variant={TextVariant.bodySm}>
+                          {
+                            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                            // @ts-ignore: Expected 0-1 arguments, but got 2.
```

Comment:
If we change this:

`const t = useContext(I18nContext);`

to this:

`const t: ReturnType<typeof useI18nContext> = useContext(I18nContext);`

I think we can get around having to use this `@ts-ignore`

Reply:
Totally, thnx 🙏